source/pipeWith.js

```diff
@@ -21,8 +21,14 @@ import identity from './identity.js';
  * @param {Function} transformer The transforming function
  * @param {Array} functions The functions to pipe
  * @return {Function}
- * @see R.composeWith, R.pipe
- * @example
+ * @see R.andThen, R.composeWith, R.pipe
+ * @examples
+ *
+ *      const doubleFn = (req) => Promise.resolve(req * 2)
+ *      const x = R.pipeWith(
```

Comment:
I believe the first argument to `pipeWith` should be `andThen`. Also we'll still have a promise here, so it's not really gonna result in just `16`. Maybe something like

```
const doubleFn = (req) => Promise.resolve(req * 2)
R.pipeWith(R.andThen, [doubleFn, R.inc])(8).then(console.log)
// logs 17
```

Reply:
Again, my comment above explains my understanding and I'll use this example directly. Please confirm I understand.